toolbox/cluster/set_scale.sh

```diff
@@ -36,18 +35,30 @@ fi
 source ${THIS_DIR}/../_common.sh
 
 INSTANCE_TYPE=${1}
-ANSIBLE_OPTS="${ANSIBLE_OPTS} -e machineset_instance_type=${INSTANCE_TYPE}"
-
+shift 
+SCALE=${1}
 shift
+if [ $# -ne 0 ]; then
+    if [ "${1}" == "--base-machineset" ]; then
+        shift
+        BASE_MACHINESET=${1}
```

Comment:
this fails if the base-machineset value isn't set:
```
./toolbox/cluster/set_scale.sh: line 44: 1: unbound variable
```
can you add a `-z ${1:-}`check + message + exit if missing?
these manual and adhoc checks will go away once we move to Python, but meanwhile .... !